pkg/controller/registry/grpc/source_test.go

```diff
@@ -84,21 +86,19 @@ func TestConnectionEvents(t *testing.T) {
 	test := func(tt testcase) func(t *testing.T) {
 		return func(t *testing.T) {
 			// start server for each catalog
-			totalEvents := 0
 			addresses := map[registry.CatalogKey]string{}
 
-			for catalog, events := range tt.expectedHistory {
-				totalEvents += len(events)
+			for catalog, _ := range tt.expectedHistory {
```

Comment:
It looks like the sanity check is complaining that omitting the keys value variable is unnecessary: 
```suggestion
			for catalog := range tt.expectedHistory {
```

Reply:
Thanks!  I update it.